src/lib/geolocation.test.ts

```diff
@@ -29,3 +29,30 @@ describe('getCountryName', () => {
         });
     });
 });
+
+describe('addRegionIdToSupportUrl', () => {
+    const originalUrl = 'https://support.theguardian.com/contribute';
```

Comment:
What do you think about adding a final test case where the `originalUrl` _doesn't_ match the pattern we expect? Just to make it clear we won't attempt to modify in that case.

Reply:
Yeah that's a good idea. I've added one.